kodilla-hibernate/src/main/java/com/kodilla/hibernate/manytomany/Employee.java

```diff
@@ -71,4 +78,14 @@ private void setLastname(String lastname) {
     public void setCompanies(List<Company> companies) {
         this.companies = companies;
     }
+
+    @Override
+    public String toString() {
+        return "Employee{" +
+                "id=" + id +
+                ", firstname='" + firstname + '\'' +
+                ", lastname='" + lastname + '\'' +
+                ", companies=" + companies +
```

Comment:
to bym usunął z `toString()`, podobnie `employees` w klasie `Company` bo może powodować `StackOverflowError` jak odpali się `toString()` na którejś z tych klas